### DISCUSSION TITLE
How we can get user's city from web api?

### DISCUSSION THREAD

**Discussion by Alex2021cyberhuman** [skoruba/IdentityServer4.Admin#893] [Q&A]
I know about city claim, but we need get user`s (not current logged user) city in client application (eg user profile). 
Thanks.

**Comment by Alex2021cyberhuman** [comment]
I can add new property to User entity. But i think it isn`t good solution.

**Reply by skoruba** [reply]
What exactly do you need - the city claim of logged user?
